lib/galaxy/model/__init__.py

```diff
@@ -4593,6 +4593,7 @@ def copy_to_collection(self, collection, destination=None, element_destination=N
                 )
             else:
                 new_element_object = element_object.copy(flush=flush)
+                new_element_object.visible = False
```

Comment:
@assuntad23 this is the complementary fix for existing collections. This sets all member HDAs of a newly copied collection to hidden. I can't think of a situation where they should ever remain visible.

Reply:
Thanks Marius!